modules/nixos/panel.nix

```diff
@@ -4,22 +4,29 @@ self: {
   lib,
   ...
 }: let
-  inherit (lib.modules) mkIf mkForce mkMerge;
+  inherit (lib.modules) mkIf;
   inherit (lib.options) mkOption mkEnableOption;
-  inherit (lib.types) attrsOf oneOf str bool int;
+  inherit (lib.types) attrsOf oneOf str bool int enum;
+  inherit (lib.lists) optionals;
 
-  pterodactlyPhp81 = pkgs.php81.buildEnv {
+  pterodactylPhp83 = pkgs.php83.buildEnv {
     extensions = {
       enabled,
       all,
     }:
-      enabled ++ (with all; [redis xdebug]);
+      enabled
+      ++ (with all; [
+        redis
+        xdebug
+      ]);
```

Comment:
Is this a nixfmt artifact?

Reply:
`xdebug` is only used during development, it can be removed here.